src/components/theme/modal/modal.service.tsx

```diff
@@ -61,6 +61,7 @@ import {
 class ModalServiceType {
 
   panel: ModalPresenting | null = null;
+  private topInsets: boolean = false;
```

Comment:
let's use something more applicable to boolean:
isTopInsetsEndabled / shouldUseTopInsets